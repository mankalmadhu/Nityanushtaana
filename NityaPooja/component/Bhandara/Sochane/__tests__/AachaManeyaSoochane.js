import React from 'react';
import ಆಚಮನೀಯಸೂಚನೆ from '../AachaManeyaSoochane';
import Renderer from 'react-test-renderer';

jest.mock('../../../Vishaya/Vidhana',()=>'ವಿಧಾನ');

test("ಆಚಮನೀಯಸೂಚನೆ" , () => {

  expect(Renderer.create(
    <ಆಚಮನೀಯಸೂಚನೆ/>
   ).toJSON()).toMatchSnapshot();

});