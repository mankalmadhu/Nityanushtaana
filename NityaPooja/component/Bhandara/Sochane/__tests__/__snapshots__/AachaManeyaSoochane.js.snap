// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಆಚಮನೀಯಸೂಚನೆ 1`] = `
<ವಿಧಾನ>
  (ಪೂರ್ವ/ಉತ್ತರ ಅಭಿಮುಖವಾಗಿ ಕುಳಿತು ಆಚಮನ ಮಾಡುವುದು. ಶೇಷ ಜಲವನ್ನು ವಿಸರ್ಜಿಸುವುದು.)
</ವಿಧಾನ>
`;
